x-pack/plugins/fleet/server/types/rest_spec/settings.ts

```diff
@@ -22,6 +22,9 @@ export const PutSettingsRequestSchema = {
         },
       })
     ),
+    has_seen_add_data_notice: schema.maybe(schema.boolean()),
+    additional_yaml_config: schema.maybe(schema.string()),
+    // Deprecated not used
```

Comment:
what's the reason for keeping these in the schema if they're not used?

Reply:
we can probably remove it, I wanted to avoid to break the API

Reply:
this is the time to make breaking changes :)